_includes/_sme-selection-checklist.html

```diff
@@ -0,0 +1,120 @@
+<h3>Choosing SMEs</h3>
+<p>
+  To assess an estimated 100 applicants, you’ll need around eight SMEs, assuming you’ll assign one SME to conduct each phone interview. To choose SMEs, work with the hiring manager to find the best-suited SMEs to participate in your hiring action. Potential SMEs should complete a background information form and submit a resume as part of the selection process. Document this information in the case file for the hiring action.
+</p>
+
+<form name="sme-selection-checklist" class="smeqa-sme-selection-checklist">
+  <h4>Qualities</h4>
+  <ul class="smeqa-sme-selection-checklist__list">
+    <li class="smeqa-sme-selection-checklist__item">
+      <label class="smeqa-sme-selection-checklist__label">
+        <input type="checkbox" name="sme1" />
+        Organizational ambassador of knowledge area.
```

Comment:
Does this also need <span class="smeqa-sme-selection-checklist_container"> </span>? I ask only because the other SME qualities have this line.

Reply:
Good catch. I got distracted by something else and missed this one.